user-service/pom.xml

```diff
@@ -278,6 +278,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>it.ozimov</groupId>
+                <artifactId>yaml-properties-maven-plugin</artifactId>
+                <version>1.1.3</version>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>read-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <files>
+                                <file>${project.basedir}/../charts/${project.artifactId}/values.yaml</file>
+                            </files>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
Is there a reason this has been duplicated? It's the same as the plugin above.

Reply:
No, should not be there.  Have removed it.